docs/HowToFAQ/FAQNuget.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Any plans to release the language server as a NuGet package? Seems like it’s not part of the Dafny release.
+---
+
+## Question
+
+Any plans to release the language server as a NuGet package? Seems like it’s not part of the Dafny release.
+
+## Answer
+
+It is now.
+Some material is on nuget ([https://www.nuget.org/packages?q=dafny](https://www.nuget.org/packages?q=dafny)):
+- The dafny tool
+- The Dafny pipeline as a library
+- The Dafny language server
+- The Dafny report generator
```

Comment:
This seems like it should be part of the docs, rather than an FAQ

Reply:
Revised